pkgs/development/python-modules/qcodes/default.nix

```
@@ -147,6 +147,11 @@ buildPythonPackage rec {
     "qcodes"
   ];
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail '"0.0"' '"${version}"'
```

```suggestion
      --replace-fail 'default-version = "0.0"' 'default-version = "${version}"'
```

nit: be more specific in the replace